cfgtool

```diff
@@ -222,14 +222,8 @@ add_user() {
   )
```

Comment:
This entire commit should be squashed. You may also consider separating the last commit into (1) Change the compressor and add crypto and (2) add the file list_clients/load_clients feature.